Dockerfile

```diff
@@ -8,19 +9,19 @@ RUN set -x && \
     # See also https://github.com/gliderlabs/docker-alpine/issues/11
     wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub && \
     wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.29-r0/glibc-2.29-r0.apk && \
-    apk add glibc-2.29-r0.apk && \
+    apk add --no-cache glibc-2.29-r0.apk && \
     rm glibc-2.29-r0.apk && \
     apk del --purge .deps
 
 RUN set -x \
     # Install ngrok (latest official stable from https://ngrok.com/download).
- && apk add --no-cache curl \
- && curl -Lo /ngrok.zip https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip \
- && unzip -o /ngrok.zip -d /bin \
- && rm -f /ngrok.zip \
+    && apk add --no-cache curl \
```

Comment:
Is `curl` still needed, now it has been removed from the line that fetches the ngrok zip? I haven't tested this, but I can't find anything obvious in the ngrok documentation that suggests it's necessary.

Reply:
That's a good point. I rebuilt the image, and everything appears to work fine without curl. I've committed the change.